spec/features/user_manages_own_mileages_spec.rb

```diff
@@ -0,0 +1,87 @@
+require "spec_helper"
+
+feature "User manages their own mileage" do
+	let(:subdomain) { generate(:subdomain) }
+  let(:user) { build(:user) }
+
+	before(:each) do
+  	create(:account_with_schema, subdomain: subdomain, owner: user)
+  	sign_in_user(user, subdomain: subdomain)
+  end
+
+  scenario "views their mileages overview" do
+    2.times { create(:mileage, user: user) }
+    click_link I18n.t("navbar.entries")
+
+    expect(page.title).to eq("#{user.full_name} | Hours")
+    expect(page).to have_content("#{user.first_name}'s entries")
```

Comment:
have_content() doesn't need the exact string right? so `user.first_name` should be enough